src/fsharp/TypeChecker.fs

```diff
@@ -10841,7 +10841,7 @@ and TcAttribute canFail cenv (env: TcEnv) attrTgt (synAttr: SynAttribute)  =
                 let tdef = tcref.ILTyconRawMetadata
                 let tref = cenv.g.attrib_AttributeUsageAttribute.TypeRef
                 
-                match TryDecodeILAttribute cenv.g tref tdef.CustomAttrs with 
+                match TryDecodeILAttribute cenv.g tref (tdef.GetFilteredCustomAttributes(cenv.g.ilg)) with 
```

Comment:
I think this call can just be `CustomAttrs` still, no need to filter when looking for `AttributeUsageAttribute`

Reply:
Yes, it should just be `CustomAttrs`.